### PR TITLE
Antwerp Branding Core Guide

### PR DESCRIPTION
**Pull request by lievenvg**
For review in testproject.

### CODE REVIEW

File: CUSTOM_BRANDING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Custom Antwerp Branding Core Guide
+If you need a slightly different version of the Antwerp Branding Core for your project, you can easily create a custom version of it. Please take a moment and read through the following guidelines to get you started.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING.md]: Focus on custom color schemes instead of custom 'anything'.

File: CUSTOM_BRANDING.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+##  Fork the Antwerp Branding Core
+The first step is to fork your own copy of the Antwerp Branding Core.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING.md]: Suggestion to make it a bit easier to read:
```
## title

tldr; paragraph

* bulleted list
* with concrete actions
```

File: CUSTOM_BRANDING.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+You can find a Fork button in the right upper corner of the [Antwerp Branding Core GitHub Page](https://github.com/a-ui/core_branding_scss) just for that.
+For more information about forking a repository, you can check the [GitHub documentation](https://help.github.com/articles/fork-a-repo).
+
+Don’t forget to update the repository name of your forked version so there can be no confusion on which repository you’re working on.
+As a naming convention we suggest to use an all lowercase snake case, like `customname_branding_scss`.
+
+##  Update files
+###  Update files in the styles folder
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING.md]: Unnecessary subtitle IMHO

File: CUSTOM_BRANDING.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+You start by updating the `main.scss` file with the files that you want to customize.
+
+```
+@import 'quarks';
+```
+
+Typically you will import a custom version of `_quarks.scss` but if you want to do other modifications like changing the font or let's say a button then you will need to modify files like `_base.scss` or `_atoms.scss`.
+Have a look at other custom versions of the Branding Core like the [ACPaaS Branding](https://github.com/a-ui/acpaas_branding_scss) for some inspiration.
+
+####  _quarks.scss
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING.md]: In Markdown, it is good practice to leave an empty line between all paragraphs (some readers, like the one I'm using right now 😉) have problems to parse the MD correctly when omitting these).

File: CUSTOM_BRANDING.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+##  Publishing
+###  Publish on npm
+
+Build the documentation for your project and commit the updated documentation.
+```
+gulp docs
+git add . && git commit -m "Updated docs"
+```
+
+Create the version number for your project and be sure to use [SemVer](http://semver.org/) for versioning. So choose a patch, a minor or a major version.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING.md]: As said earlier, I would omit this part, but if you prefer, you can provide this information in the form of some useful links.

File: CUSTOM_BRANDING.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+####  _quarks.scss
+The quarks contain variables for colors, spacing, etc.
+If you want to override some variables, you just import your custom version of the `quarks.variables`.
+
+```
+@import 'quarks/quarks.variables';
+```
+
+###  Remove unnecessary files from `styles` folder
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING.md]: Where's the part where you import the other atomic styles from the 'node_modules'? Also, I have the feeling that some steps are in the wrong logical order. More logically (opinionated) would be:
- fork core branding
- add core branding to dev dependencies
- create an empty `main.scss` where you:
        - override your variables
        - import the atomic parts from the added core branding
- update 'necessary evil' like readmes and changelogs
- create build
- extras:
        - point to the fact that you can override other atomic parts too
        - ref. links to go even further (publishing, CDN, …)

File: CUSTOM_BRANDING_DUTCH.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
+# Aanpassen van Antwerp Branding Core kleurpatronen
+
+Dit document beschrijft bondig hoe je een kleurvariant van de Antwerp Branding Core kan maken.
+
+## Fork de Antwerp Branding Core
+
+* Fork je versie van de [Antwerp Branding Core](https://github.com/a-ui/core_branding_scss).
+* Meer informatie over het ‘forken’ van een repository vind je in de [Git documentatie](https://git-scm.com/book/nl/v2/GitHub-Aan-een-project-bijdragen).
+
+Vergeet niet om de naam van je project aan te passen zodat er geen verwarring kan ontstaan met de oorspronkelijke projectnaam. Voor de naamgeving van je project stellen we voor om dit in `Snake case` met kleine letters te schrijven, bv.: `customname_branding_scss`.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING_DUTCH.md]:
```suggestion
Vergeet niet om de naam van je project aan te passen zodat er geen verwarring kan ontstaan met de oorspronkelijke projectnaam. Voor de naamgeving van je project kan je best `snake case` met kleine letters gebruiken, bv.: `customname_branding_scss`.
```

File: CUSTOM_BRANDING_DUTCH.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+## Update bestanden
+
+### package.json
+
+* Pas de `package.json aan met de project naam, versienummer en de url van de repository.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING_DUTCH.md]:
```suggestion
* Pas de `package.json` aan met de project naam, versienummer en de url van de repository.
```

File: CUSTOM_BRANDING.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+@import '@a-ui/core/dist/assets/styles/base';
+@import '@a-ui/core/dist/assets/styles/utilities';
+@import '@a-ui/core/dist/assets/styles/atoms';
+@import '@a-ui/core/dist/assets/styles/molecules';
+@import '@a-ui/core/dist/assets/styles/organisms';
+
+@import '@a-ui/core/dist/assets/styles/aui';
+```
+
+### _quarks.scss
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING.md]:
```suggestion
### \_quarks.scss
```

File: CUSTOM_BRANDING_DUTCH.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+@import '@a-ui/core/dist/assets/styles/base';
+@import '@a-ui/core/dist/assets/styles/utilities';
+@import '@a-ui/core/dist/assets/styles/atoms';
+@import '@a-ui/core/dist/assets/styles/molecules';
+@import '@a-ui/core/dist/assets/styles/organisms';
+
+@import '@a-ui/core/dist/assets/styles/aui';
+```
+
+### _quarks.scss
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING_DUTCH.md]:
```suggestion
### \_quarks.scss
```

File: CUSTOM_BRANDING_DUTCH.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+@import 'quarks/quarks.variables';
+
+@import '@a-ui/core/dist/assets/styles/quarks/quarks.variables';
+```
+
+### CHANGELOG.md
+
+* Vervang het originele `CHANGELOG.md bestand met een aangepaste versie.
+
+Voor de formatering volg je best de [Keep a Changelog](http://keepachangelog.com/) richtlijnen en pas [semantisch versiebeheer](https://semver.org/)  op je project toe.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING_DUTCH.md]:
```suggestion
Voor de formattering volg je best de [Keep a Changelog](http://keepachangelog.com/) richtlijnen en pas je [semantisch versiebeheer](https://semver.org/) op je project toe.
```

File: CUSTOM_BRANDING_DUTCH.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+### README.md
+
+* Pas het originele `README.md` bestand aan met project gerelateerd informatie.
+
+### Remove unnecessary files
+
+* Verwijder de `scss` bestanden, die je niet hebt aangepast, van de styles map.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING_DUTCH.md]:
```suggestion
* Verwijder de `scss` bestanden, die je niet hebt aangepast, uit de styles map.
```

File: CUSTOM_BRANDING_DUTCH.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+## Build
+
+* Build het project via het `gulp build` commando.
+* Build de documentatie app via het `gulp docs` commando.
+
+## Extra
+
+* Volg de [npm instructies](https://docs.npmjs.com/cli/publish) om het pakket op de npm te publiceren.
+
+Meestal zal je enkel een aangepaste versie van het `_quarks.scss` bestand moeten importeren, maar als je andere aanpassingen wil doen, zoals bijvoorbeeld de font of een knop aanpassen, dan moet je ook de `_base.scss` of `_atoms.scss` bestanden aanpassen.
```
Comment (TriangleJuice) [inline CUSTOM_BRANDING_DUTCH.md]:
```suggestion
Meestal zal je enkel een aangepaste versie van het `_quarks.scss` bestand moeten importeren, maar als je andere aanpassingen wil doen, zoals bijvoorbeeld het font of het design van een knop, dan moet je ook de `_base.scss` of `_atoms.scss` bestanden aanpassen.
```
